app/middlewares/token/checkToken.js

```diff
@@ -0,0 +1,16 @@
+import jwt from 'jsonwebtoken';
+import { db_config } from '../../config/db_config';
+
+export const checkToken = (req, res, next) => {
+  if (req.path === '/login') return next();
+
+  const token = req.headers['x-access-token'];
+
+  if (!token) return res.status(401).send({ message: 'No token provided' });
+
+  return jwt.verify(token, db_config.jwt_secret, err => {

```

Comment:
`jwt_secret` is not related to db_config. Move it to another config or rename db_config.

Reply:
Fixed